### PR TITLE
double cock moyai?

### PR DESCRIPTION
**Pull request by ImVerum**

### CODE REVIEW

File: Modules/UtilityCommands.cs [hunk 1]
```diff
@@ -25,9 +25,9 @@ public async Task GetRoundPFP(SocketUser user)
         }
         [Command("cockmoyai")]
         [Summary("cock moyai")]
-        public async Task CockMoyai(SocketUser user)
+        public async Task CockMoyai()
```
Comment (Jan0660) [inline Modules/UtilityCommands.cs:28]: yes

Comment (ImVerum) [reply Modules/UtilityCommands.cs:28]: did i do it right this time?

File: Modules/UtilityCommands.cs [hunk 2]
```diff
@@ -25,9 +25,9 @@ public async Task GetRoundPFP(SocketUser user)
...
         {
-            await ReplyAsync("cock moyai🗿🗿🗿");
+            await ReplyAsync("cock moyai")
```
Comment (Jan0660) [inline Modules/UtilityCommands.cs:30]: kinda gay bro?

Comment (ImVerum) [reply Modules/UtilityCommands.cs:30]: tf i didnt remove that

Comment (ImVerum) [reply Modules/UtilityCommands.cs:30]: frick github i guess

Comment (Jan0660) [reply Modules/UtilityCommands.cs:30]: apparetnly you did

Comment (ImVerum) [reply Modules/UtilityCommands.cs:30]: tf

File: Modules/UtilityCommands.cs (hunk 1 above)
Comment (ImVerum) [reply Modules/UtilityCommands.cs:28]: appreciated

Comment (ImVerum) [reply Modules/UtilityCommands.cs:28]: ;

**Review by ImVerum — COMMENTED** [review]
;;;;;

File: Modules/UtilityCommands.cs (hunk 1 above)
Comment (Arcer) [reply Modules/UtilityCommands.cs:28]: EA

Comment (Jan0660) [reply Modules/UtilityCommands.cs:28]: no

Comment (Arcer) [reply Modules/UtilityCommands.cs:28]: It's in the game

Comment (Arcer) [reply Modules/UtilityCommands.cs:28]: No u

**Pull request merged**